hack/perf/README.md

```diff
@@ -0,0 +1,10 @@
+# A set of scripts to compare the performance of two container engines
+
+Run the scripts via `sudo sh $script.sh`.
+
```

Comment:
Maybe worth to point out that this will run `system prune` so user are aware that they have to backup their containers/data if needed.

Reply:
Nice ide!